main.go

```diff
@@ -15,6 +18,88 @@ import (
 var telemetryDB = make(map[string]TelemetryData)
 var listIPs = make(map[int]string)
 
+// Request is either:
+//   - Payload Ops (Ground)
+//   - Uplink/Downlink (Ground)
+func handleScenarioOne(c *gin.Context, r RedirectRequest) {
```

Comment:
This isn't the biggest deal in the world, but I feel like the scenario function names could be more descriptive 😅